addons/music/fnc_getMusicPath.sqf

```diff
@@ -0,0 +1,28 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getMusicPath
+Description:
+    A function used to return the config path of a music file
+Parameters:
+    one string
+Example:
+    (begin example)
+    _configPath = ["LeadTrack01_F_Bootcamp"] call CBA_fnc_getMusicPath
+    (end example)
+Returns:
+    config
+Author:
+    Fishy
+---------------------------------------------------------------------------- */
+#include "script_component.hpp"
+
+private ['_vars','_config','_duration'];
+if (!params [["_className",'']]) exitWith {WARNING('No Class name given');};
+if (typeName _className != 'STRING') exitWith {WARNING_1('Class not a string',_className);};
```

Comment:
You could combine L19 and L20 into one line.
e.g.: `If !(params [["_classname","",[""]]]) exitWith {WARNING('no classname was provided');};`